config/prow/cluster/secretrotator-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: default
```

Comment:
Can you add a `Namespace` resource to this PR and move everything into a `k8s-gsm-tools` namespace?

Reply:
I put both deployments and the service account into the `k8s-gsm-tools` namespace.
ClusterRole is not namespaced, so I left it as it was.